.circleci/config.yml

```diff
@@ -6,13 +6,13 @@ commands:
       - restore_cache:
           name: Restore Yarn Cache
           keys:
-            - yarn-i18n-v3-{{ .Environment.CIRCLE_JOB }}-{{ checksum "yarn.lock" }}
+            - yarn-i18n-v4-{{ .Environment.CIRCLE_JOB }}-{{ checksum "yarn.lock" }}
```

Comment:
Try to fix the UI Test CI failing by clearing cache but it didn't work.
Now that the cache folder was created, not sure if we should roll it back.